uint8 PREEMPTED = 0
uint8 SUCCEEDED = 1
uint8 NO_IK = 2
uint8 OBSTRUCTED = 3
uint8 NOT_EXECUTING = 4
uint8 EXECUTING = 5
uint8 CONDITION_ERROR = 6
uint8 status